data/DataType/bool_python.yaml

```diff
@@ -0,0 +1,7 @@
+keyword: bool
+integer-min: '0'
+integer-max: '1'
+integer-signed: false
+literals:
+  - 'true'
```

Comment:
This can be True and False. Rest looks good to me.

Reply:
quotes not needed.

must be `True` and `False`.

python doesnt have `true` or `false`